### DISCUSSION TITLE
Changing Pronouns for Jailbreak

### DISCUSSION THREAD

**Discussion by PeronGH** [Richard-Weiss/node-chatgpt-api#9] [General]
In the past, we used a 2nd-person jailbreak message sent by `[system](#additional_instructions)`. However, I noticed that it does not seem as effective as it used to be. Recently, a Redditor successfully leaked the latest [initial prompt](https://www.reddit.com/r/bing/comments/18a15q7/bings_initial_prompt_as_of_december_2023/), revealing that Microsoft has changed the pronoun in the initial prompt. I also asked the jailbroken Sydney about this; it [says](https://sharegpt.com/c/8KKMWSa) that the initial prompt was sent by itself (`[assistant](#instructions)`) and it believes the `[system](#additional_instructions)` is sent by someone else.

Can someone else investigate this? And should we move to the 1st person jailbreak message?

**Comment by Richard-Weiss** [comment]
@PeronGH You can choose the system message yourself by including it in the request body and/or `.env`.
You always have to negate the old one so it will be more accepting of it. 
But messages like these are usually sent by the system, just like the inner_monologue.
Bing doesn't really know much about itself and hallucinate quite a lot in that regard.
However I found that when Turbo was working that it could see the new system message but chose to ignore it.
I've tried the system message from the `.env.example` and it works just fine, or is there something specific you have a problem with?

**Reply by PeronGH** [reply]
I'm currently using `SydneyQt` with `sydney_webui` that I wrote myself. I'm just curious about my findings.

According to [an early leak of the initial prompt](https://www.reddit.com/r/bing/comments/11bd91j/release_of_the_whole_initial_prompt_of_bing_chat/), `inner_monologue` is sent by the assistant itself.

Although Bing may not know much about itself, it should be able to repeat what Microsoft has programmed into its context. The creative mode in Bing seems to consistently say that the instruction starts with "I," while the precise mode or GPT-4 Turbo indicates it starts with "You."

Considering the recently leaked prompt, I think there might have been a change in the pronoun used for the creative mode.

**Reply by PeronGH** [reply]
Also, I have already changed the jailbreak system message in my `sydney_webui` to use `[assistant](#instructions)` and the first-person pronoun. The result is quite promising. However, I guess more tests and investigation may be needed. I'm wondering what your thoughts are on this?

**Reply by Richard-Weiss** [reply]
@PeronGH I'm not sure. What difference do you see?

**Reply by PeronGH** [reply]
I've found the jailbreak to be more consistent and effective after changing the pronoun and sender. It seems that, for Copilot, maintaining consistency with what it has said itself is easier than following instructions from the system.

**Reply by Richard-Weiss** [reply]
@PeronGH I can try creating some repeatable tests tomorrow to account for the LLM non deterministic output and see how both methods differ. It would be helpful to have a specific system message from you, because I'm not having any trouble whatsoever with the current implementation.

**Reply by PeronGH** [reply]
The old one was still working for me, too. However, I just found that the new one works better.

Here is the commit where I changed the pronoun: https://github.com/PeronGH/sydney_webui/commit/0d1eddffdb3bf1770c27a995cad53ad7cc42e7bd.

**Reply by Richard-Weiss** [reply]
@PeronGH Sorry, but I'm a bit confused. I know how I would do it for this repo, I would just have to change `system` to `assistant` for the prompt injection. I meant more like how the other method would be different in an objective sense. I'll just play around a bit with it tomorrow and try out some different system messages with it.
